aws/cloudformation/moz-single.json

```diff
@@ -33,6 +33,16 @@
       "Type": "String",
       "Description" : "subdomain"
     },
+    "VPC": {
+      "Description": "The VPC into which the stack will be deployed",
+      "Type": "String",
+      "Default": "vpc-9037f3f5"
```

Comment:
I pulled these magic values from the AWS dev console, not sure if there's any way to discover them automagically.